src/main/java/org/simplify4u/plugins/CheckMojo.java

```diff
@@ -175,6 +175,9 @@ public static class VerificationResult {
     @Parameter(property = "pgpverify.reportWrite", defaultValue = "false")
     private boolean reportWrite;
 
+    @Parameter(property = "pgpverify.failDuplicateKeyItem", defaultValue = "false")
```

Comment:
Please add a documentation for new parameter with since tag, as new feature will be 1.19.0